.gitignore

```diff
@@ -40,3 +40,6 @@ pkg
 #
 # For vim:
 #*.swp
+
+# deps should be good enough in Gemfile and gemspec that this isn't needed
+Gemfile.lock
```

Comment:
I disagree. 
Pushing Gemfile.lock to git is a better option IMO, because it makes easier to git bisect issues, as you would use the same dependencies at that point in time.